### PR TITLE
PhenoPLIER on Asthma-COPD Overlap Syndrome: add base files for project

### PR DESCRIPTION
**Pull request by miltondp**
This PR adds the base to start working on this project to apply PhenoPLIER on Asthma-COPD Overlap Syndrome (ACOS).

### CODE REVIEW

File: libs/utils.py [hunk 1]
```diff
@@ -173,3 +173,23 @@ def _remove_next_suffix(f):
         filepath = filepath.with_suffix("")
 
     return filepath
+
+
+def read_log_file_and_check_line_exists(log_file, expected_lines: list):
```
Comment (jjc2718) [inline libs/utils.py:178]: A bit of a nit pick, but these aren't really "expected lines", are they? (And it's not totally clear just from the function definition what a "line" is - is it a number? A string?)

Maybe a variable name like `expected_prefixes` would be clearer than `expected_lines` - to me, a list of "expected prefixes" is more clearly a list of strings that you expect to find at the start of log file lines, which it looks like is what you're intending.

File: projects/asthma-copd/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# PhenoPLIER analyses of Asthma-COPD Overlap Syndrome (ACOS)
```
Comment (jjc2718) [inline projects/asthma-copd/README.md:1]: Do you want to fill this file out in more detail, or will you do that in a future PR?

**Review by jjc2718 — APPROVED** [review]
Looks good! Just a few small comments.

File: libs/utils.py (hunk 1 above)
Comment (miltondp) [reply libs/utils.py:178]: Fantastic! I would have the same question in the future :smiley:

File: projects/asthma-copd/README.md (hunk 2 above)
Comment (miltondp) [reply projects/asthma-copd/README.md:1]: Yes, I will fill it out in the future, it doesn't make much sense to do that now. I've updated it though with some minor changes.

**Pull request merged**